katib/components/katib-db-mysql/kustomization.yaml

```diff
@@ -10,3 +10,6 @@ images:
   - name: mysql
     newTag: "8"
     newName: mysql
+commonLabels:
+  app.kubernetes.io/component: katib
+  app.kubernetes.io/name: katib-controller
```

Comment:
Is this the right name? Did you mean to call everything katib-controller even if its a db?

Reply:
Currently, it works like this. 
We have one Application: https://github.com/kubeflow/manifests/blob/master/katib/katib-controller/overlays/application/application.yaml, for katib-controller, katib-db-manager and katib-mysql with this label (https://github.com/kubeflow/manifests/blob/master/katib/katib-controller/overlays/application/kustomization.yaml#L4).
If we want to make unique label for each component, do we need to create individual application for them?